ush/rocoto/setup_workflow_fcstonly.py

```diff
@@ -653,13 +654,14 @@ def get_workflow(dict_configs, cdump='gdas'):
         tasks.append('\n')
 
     # vrfy
-    deps = []
-    dep_dict = {'type':'metatask', 'name':f'{cdump}post'}
-    deps.append(rocoto.add_dependency(dep_dict))
-    dependencies = rocoto.create_dependency(dep=deps)
-    task = wfu.create_wf_task('vrfy', cdump=cdump, envar=envars, dependency=dependencies)
-    tasks.append(task)
-    tasks.append('\n')
+    if do_vrfy in ['Y', 'YES']:
```

Comment:
See my other comment about the need for this toggle.

Reply:
Answered.